src/deserialize.zig

```diff
@@ -6,8 +6,9 @@ const eql = std.mem.eql;
 const readIntSliceBig = std.mem.readIntSliceBig;
 const ArrayList = std.ArrayList;
 const hasFn = std.meta.trait.hasFn;
+const Allocator = std.mem.Allocator;
 
-const implementsDecodeRLP = hasFn("decodeRLP");
+const implementsDecodeRLP = hasFn("decodeFromRLP");
```

Comment:
I did this renaming since the encoder is named `encodeToRLP` -- to be coherent.
We could also remove the `To` in the encoder and have `encodeRLP` and `decodeRLP` but I think I like most this style.